modules/candidate_parameters/ajax/formHandler.php

```diff
@@ -363,71 +363,68 @@ function editConsentStatusFields($db, $user)
         exit;
     }
 
+    // get CandID
     $candIDParam = $_POST['candID'];
     $candID      = (isset($candIDParam) && $candIDParam !== "null") ?
         $candIDParam : null;
 
-    $id = null;
-    if (!(is_null($_SESSION['State']))) {
```

Comment:
why get rid of this check?

Reply:
good question; i have no idea. @ridz1208 made this change initially - rida, do you know why?

Reply:
@ZainVirani we're replacing grabbing the user from the session variable with user singleton which doesn't need a check for whether it's set